__tests__/app.test.js

```diff
@@ -113,4 +113,59 @@ describe("GET/not-a-route", () => {
           });
         });
       });  
-    });
\ No newline at end of file
+    });
+describe("GET /api/articles/:article_id/comments", () => {
+    it("responds with an array of comments for the given article_id of which each comment should have the correct properties", () => {
+        const articleId = 1
+        return request(app)
+            .get(`/api/articles/${articleId}/comments`)
+            .expect(200)
+            .then((res) => {
+                const { body } = res;
+                const { comments } = body;
+                expect(comments.length).toBeGreaterThan(0);
+                comments.forEach((comment) => {
+                    expect(comment).toHaveProperty("comment_id", expect.any(Number));
+                    expect(comment).toHaveProperty("votes", expect.any(Number));
+                    expect(comment).toHaveProperty("created_at", expect.any(String));
+                    expect(comment).toHaveProperty("author", expect.any(String));
+                    expect(comment).toHaveProperty("body", expect.any(String));
+                    expect(comment).toHaveProperty("article_id", expect.any(Number));
+                });
+                const sortedComments = comments.sort((a, b) => a.created_at >= b.created_at ? 1 : -1);
+                expect(comments).toEqual(sortedComments)
+            });
+    });
+    it("404: responds with an error if article_id is not found", () => {
```

Comment:
This test is testing another endpoint. It should be 'GET /api/articles/:article_id/comments'.

Reply:
resolved